### PR TITLE
Ysp 321  profile vertical images

### PR DESCRIPTION
**Pull request by joetower**
### Description of work
- Adds dial option for portrait oriented images in the Profile Banner, - `portrait` or `landscape`
- Adds dial option for alignment - `left` or `right`
- Adds dial option for image style - `inline` or `outdent`

### Testing Link(s)
- [x] Test in component library here: https://github.com/yalesites-org/component-library-twig/pull/362

### Functional Review Steps
- [ ] Navigate to: https://pr-642-yalesites-platform.pantheonsite.io/profile/alyssa-dechiaro
- [ ] Verify the image in the banner is `portrait` in orientation and styled with the `outdent` option
Screenshot 2024-05-07 at 14 31 31

- [ ] If you login and edit the profile, verify that you can: 
  - [ ] Change the image orientation - `portrait` or `landscape`
  - [ ] Change the image alignment - `left` or `right`
  - [ ] Change the image style - `inline` or `outdent`

Screenshot 2024-05-07 at 14 31 02

- [ ] Test each option - `landscape` orientation, `inline` style or `right` alignment

### CODE REVIEW

**Comment by miketullo95** [comment]
howdy! this is looking great! i did notice that i dont have the options to edit the banner area with the image though, im assuming thats some sort of permissions thing, maybe? @joetower 

Screenshot 2024-05-07 at 2 50 52 PM

**Review by codechefmarc — APPROVED** [review]
I tested this out on the multidev and all worked great. However, wanted to note that I did have to click on Jim's "Update locks" and then flush cache to get this to work for an "contributor" role. After that, all edits worked fine.

File: web/profiles/custom/yalesites_profile/config/sync/core.entity_view_display.node.profile.default.yml [hunk 1]
```diff
@@ -99,6 +97,7 @@ third_party_settings:
       - 'Content fields'
       - 'Custom block types'
       - 'Custom blocks'
+      - Devel
```
Comment (codechefmarc) [inline web/profiles/custom/yalesites_profile/config/sync/core.entity_view_display.node.profile.default.yml:100]: I noticed Devel is in this file listed as a possible block that can be added? Is this OK or should be removed? @nJim

**Comment by miketullo95** [comment]
controls work now! this is great, could we have the new default for image orientation be `portrait`? would be nice for all new profiles created after this release to have this as a default @joetower

**Comment by joetower** [comment]
@miketullo95 I pushed up a fix to make `portrait` the default. Could you please take another look. When you edit the banner section on a new profile node, you should see this: 

Screenshot 2024-05-14 at 13 16 13

**Comment by miketullo95** [comment]
@joetower it looks like the drop down is correct, but the front facing image is still landscape. If i switch it to landscape and back to portrait it works. but by default its incorrect 

right after creating the profile
Screenshot 2024-05-14 at 2 38 22 PM

after toggling `landscape` -> `portrait` 
Screenshot 2024-05-14 at 2 40 16 PM

**Comment by joetower** [comment]
@miketullo95 Ah, I missed changing the default in the build function Marc wrote. Could you please give this another test? It should be correct now. Apologies for the back and forth.

**Review by miketullo95 — APPROVED** [review]
👍

File: web/profiles/custom/yalesites_profile/config/sync/core.entity_view_display.node.profile.default.yml (hunk 1 above)
Comment (nJim) [reply web/profiles/custom/yalesites_profile/config/sync/core.entity_view_display.node.profile.default.yml:100]: Removing this line. Good catch.

**Pull request merged**